<?xml version="1.0" encoding="UTF-8"?>
<alr:test-suite xmlns:alr="urn:model.allure.qatools.yandex.ru" start="1556668816222" stop="1556668816226" version="1.4.0">
  <name>Magento\Signifyd\Test\Unit\Model\MessageGenerators\PatternGeneratorTest</name>
  <test-cases>
    <test-case start="1556668816222" stop="1556668816223" status="passed">
      <name>testGenerateThrowsException</name>
    </test-case>
    <test-case start="1556668816223" stop="1556668816224" status="passed">
      <name>testGenerate with data set #0</name>
    </test-case>
    <test-case start="1556668816224" stop="1556668816225" status="passed">
      <name>testGenerate with data set #1</name>
    </test-case>
    <test-case start="1556668816225" stop="1556668816225" status="passed">
      <name>testGenerate with data set #2</name>
    </test-case>
    <test-case start="1556668816226" stop="1556668816226" status="passed">
      <name>testGenerate with data set #3</name>
    </test-case>
  </test-cases>
</alr:test-suite>
